src/magellan_core/params/odom_ekf.yaml

```diff
@@ -1,7 +1,7 @@
 # The frequency, in Hz, at which the filter will output a position estimate. Note that the filter will not begin
 # computation until it receives at least one message from one of the inputs. It will then run continuously at the
 # frequency specified here, regardless of whether it receives more measurements. Defaults to 30 if unspecified.
-frequency: 100
+frequency: 5
```

Comment:
Why was this dropped so low?

Reply:
we were trying to echo ekf information and reduced the rate so that we could read the rostopic echo better. It can be changed back to be consistent with the master branch.